fixtures/nesting/src/modern/AboutPage.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {useContext} from 'react';
+
+import ThemeContext from './shared/ThemeContext';
+import withLegacyRoot from './withLegacyRoot';
+
+const Greeting = withLegacyRoot(() => import('../legacy/Greeting'));
```

Comment:
This is the API.

Reply:
(If you build in prod mode, webpack puts the second React in a chunk, as expected.)